tox.ini

```diff
@@ -1,17 +1,22 @@
 [tox]
-envlist = flake8,py27,py34,py35
+envlist = flake8,py27,py36
 
 [testenv]
 setenv =
     CPPFLAGS=-I{toxinidir}/tmp-build/include
     LDFLAGS=-L{toxinidir}/tmp-build/lib
     C_INCLUDE_PATH={toxinidir}/tmp-build/include
     LD_LIBRARY_PATH={toxinidir}/tmp-build/lib
+    PYTHONIOENCODING = UTF-8
```

Comment:
why this?

Reply:
To ensure the default file encoding is utf-8. This caused issues on my machine after you added non-ascii characters to the integration tests.

Reply:
Even after I added this?
https://github.com/confluentinc/confluent-kafka-python/blob/master/examples/integration_test.py#L2